examples/01_example_wsiread.ipynb

```diff
@@ -12,7 +12,7 @@
     "\n",
```

Comment:
<p>The next cell should be SKIPPED on all platforms, except for Colab or Kaggle, where it installs<code>tiatoolbox</code>, and other prerequisite software. Harmless error messages should be ignored. Outside Colab or Kaggle, the notebook expects  <code>tiatoolbox</code> to already be installed. (See the instructions in <a href="https://github.com/TIA-Lab/tiatoolbox/blob/master/README.md#install-python-package" target="_blank">README</a>.)</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/TIA-Lab/tiatoolbox/pull/132/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='5'/>

Reply:
Addressed in this commit https://github.com/TIA-Lab/tiatoolbox/pull/132/commits/f63e9bef28deb14b8e74a62bb9df12806ebceedc